### ISSUE TITLE
[Accessibility] acm-html has an unlabeled image

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-journals/acm#26]
When this template is rendered to `acm-html` format, regardless of the teaser image setup in the YAML, the following image is embedded:

![image](https://github.com/quarto-journals/acm/assets/19754711/aab398fa-95d7-4f93-b13c-96e500676a49)

First issue is that this image does not have any alt text, and the second issue is that this image should not be present unless the users specify. However, I am not sure about whether this image is inserted into all the ACM proceedings. If so, please add a default alt text.

**Comment by cscheid** [comment]
That's an ORCID badge, and my understanding is that the ACM template includes ORCID information for all authors. In that sense, we're just following the ACM template on this (and so I'm not sure I agree with "this image should not be present unless the users specify"; I think we should be following what the original template does as closely as we can). 

We should make sure that the badge itself has alt text, but I'm not sure what it should be (given that we're basing it from the PDF format provided by the ACM which doesn't have any accessibility labels.)

**Comment by jooyoungseo** [comment]
@cscheid -- Aw, I see. Can we add "ORCID Badge" as the default alt text? For now, without any alt text, it is read like a random hash, which does not make sense to screen reader users.
